libc-test/build.rs

```diff
@@ -2458,7 +2449,17 @@ fn test_linux(target: &str) {
                                            field == "_pad2" ||
                                            field == "ssi_syscall" ||
                                            field == "ssi_call_addr" ||
-                                           field == "ssi_arch"))
+                                           field == "ssi_arch")) ||
+        // FIXME: After musl 1.1.24, it have only one field `sched_priority`,
+        // while other fields become reserved.
+        (struct_ == "sched_param" && [
+            "sched_ss_low_priority",
+            "sched_ss_repl_period",
+            "sched_ss_init_budget",
+            "sched_ss_max_repl",
+        ].contains(&field) && musl) ||
+        // FIXME: After musl 1.1.24, the type becomes `int` instead of `unsigned short`.
```

Comment:
This means that musl 1.1 has ABI breaking changes with respect to musl 1.0.x ?

Reply:
Yes

Reply:
@gnzlbg Oh, sorry. It may not be an **ABI breaking** change. The struct layout is kept and just some fields are renamed to reserved.

Reply:
Does that refer to the change in type of `ipc_perm.__seq` field from `unsigned short` to `int` on aarch64 ? Do you have a musl diff for this change?

Reply:
IIUC it was unsupported https://github.com/bminor/musl/commit/827aa8fbcac89a63c6efb986871663861500cd13
But still it's breaking change between 1.1.23 and 1.1.24

Reply:
Relevant commit for this change https://github.com/bminor/musl/commit/319b2d02e7799107c0090eb088be48d17df0825c